draft-ietf-git-using-github.md

```diff
@@ -428,6 +447,242 @@ A working group that uses GitHub MAY provide either facility at the request of
+
+Chairs might choose to manage the process of deciding which issues are
+substantive.  For instance, chairs might reserve the ability to use the `design`
+label to new issues (see {{label-design}}) and to close issues marked as `design`.
+Chairs should always allow document editors to identify and address editorial
+issues as they see fit.
+
+As documents mature further, explicit confirmation of technical decisions with
+the Working Group mailing list becomes more important.
+
+Gaining Working Group consensus about the resolution of issues can be done in
+the abstract, with editors being permitted to capture the outcome of discussions
+as they see fit.
+
+More mature documents require not only consensus, but consensus about specific
+text.  All substantive changes to documents that have passed WGLC MUST be
+proposed as pull requests, discussed on the mailing list, and have chairs
+explicitly confirm consensus.  Chairs MAY institute this stricter process prior
+to WGLC.
+
+Note:
+
+: It is generally sufficient to trust editors to manage adherence with these
+  policies, aided by the transparency provided by the version control system.
+  There are tools that can be used to more tightly control access to
+  repositories, but they can be overly constraining.
+
+
+## Issue Labelling Schemes {#labels}
+
+Several schemes for use of issue labels in managing issues have been used.  This
+section outlines these strategies and how they might be applied.
+
+This document suggests that only a design/editorial split is useful in all
+cases.  Working Groups that adopt the "light" usage model might not need
+additional rules around the use of labels.
+
+Working Groups or editors might use additional labels as they choose.  Any label
+that is used as part of a process requires that the process be documented and
+announced by Working Group chairs.  Editors SHOULD be permitted to use labels to
+manage issues without any formal process significance being attached to those
+issues.
+
+
+### Design/Editorial Labelling {#label-design}
+
+The most important distinction about an issue is whether it is substantive.  The
+labels of `design` and `editorial` are used to represent this distinction.
+
+An issue labeled as `design` has or might have a substantive effect on a
+document.  For protocol specifications, a `design` issue usually affects
+implementations or interoperability requirements.  Addressing a `design` issue
+ultimately requires Working Group consensus, even if the resolution is to make
+no change.
+
+An issue labeled as `editorial` has no substantive effect on a document, except
+to the extent that addressing the issue might make understanding the
+specification easier.  Resolution of `editorial` issues can be left to the
+discretion of editors.
+
+This distinction can be applied to all types of document.  For instance, a
+`design` issue for an Informational document might be raised to discuss
+important concepts.
+
```

Comment:
Hmmm, "usually affecting an implementation or interop" (above) seems pretty different from "discussing important concepts". Maybe the right distinction for an informational document is a substantive change is `design` (something that changes the meaning of the document). I may be over-thinking this.

Reply:
Over-thinking or not, it's still quite helpful.

Reply:
For a protocol spec, anything that breaks compat is design.  For an Informational spec, anything that changes what a reader might choose to do with the information is a design change.  Discussing new topics or changing position on a topic would therefore be a design change, in my mind.  But I agree, there's less of a bright line in a document that doesn't eventually lead to bits on the wire.